src/main/java/tfifteenfour/clipboard/logic/commands/CommandResult.java

```diff
@@ -9,41 +9,28 @@
  */
 public class CommandResult {
 
-    private final String feedbackToUser;
+    private final Command command;
 
-    /** Help information should be shown to the user. */
-    private final boolean showHelp;
+    private final String feedbackToUser;
 
-    /** The application should exit. */
-    private final boolean exit;
+    /** The roster state has been modified */
+    private final boolean hasChangedModelState;
```

Comment:
Is there a reason why it is `hasChangedModelState` instead of `hasChangedRosterState`?

Reply:
Interesting question. I used Model because the `LogicManager` where all the commands are processed doesn't view `Roster` directly, it has to act on the `Model` in order to modify `Roster`. In that sense, the states of our program from the `LogicManager`'s perspective are `Model`s. 

Though upon thinking about it now, it might be more intuitive to phrase it as `hasChangedRosterState`, since a `Command` only acts within the scope of a `Roster`, not the entire `Model`. Will make the change.